### PR TITLE
Dynamic front office

### PR DESCRIPTION
**Pull request by Kmaschta**

### CODE REVIEW

File: apps/api/src/index.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ app.use(bodyParser());
 app.use(error(formatError));
 
 app.use(serve(path.resolve(__dirname, '../public')));
```
Comment (alexisjanvier) [inline apps/api/src/index.js:45]: So we don't need this anymore, right?

**Review by alexisjanvier — CHANGES_REQUESTED** [review]

File: apps/api/src/front.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    const errors = validate(request);
+
+    if (errors && Object.keys(errors).length > 0) {
+        ctx.status = 400;
+        ctx.state.errors = errors;
+        ctx.state.request = request;
+        return ctx.render('index.ejs');
+    }
+
+    // TODO: Create the request in DB
```
Comment (floo51) [inline apps/api/src/front.js:71]: Let's do it here?

File: apps/api/src/index.js (hunk 1 above)
Comment (Kmaschta) [reply apps/api/src/index.js:45]: Yes we do, because of all the assets: CSS, and stuff (look at the links)

File: apps/api/src/front.js (hunk 2 above)
Comment (Kmaschta) [reply apps/api/src/front.js:71]: I would prefer to do it in another PR, this one's already huge

**Review by floo51 — APPROVED** [review]

**Pull request merged**